scripts/README.md

```diff
@@ -34,6 +34,7 @@ The following steps must be proposed as a pull request.
    1. Keep the package versions of stable public packages the same as the root `package.json` version.
    2. Don't bump private packages unless necessary, these have `"private": true` in their `package.json`
    3. Manually remove `^` from packages with prerelease version, e.g. `-alpha`
+   4. [IMPORTANT] Search for the older version string in the whole project directory and bump packages and it's references that are still on the old version.
```

Comment:
Do we know if what @mnajdova suggested in https://github.com/mui/material-ui/pull/39485#pullrequestreview-1682432699 is true?

>  I assume running `yarn release:versions` again would have picked those changes up.

In that case the manual check wouldn't be necessary, right?

Reply:
This is a general instruction for unchanged packages, in today's case, it was **utils**.

Reply:
I see, and do we know why it happened? I'm trying to figure out if there's something we can fix on our automated process before adding a manual step 😅

Reply:
This is a manual process where you need to find all packages with older version and update it to newer version. Lerna doesn't pick those. That's why I added the instructions here.